packages/sdk/src/types.ts

```diff
@@ -428,6 +428,8 @@ export type OrderFulfillment = {
   state: FulfillmentState;
   trackingNumber?: Maybe<Scalars['String']>;
   type: FulfillmentType;
+  minutesUntilReady?: Maybe<Scalars['Int']>;
```

Comment:
I think you need readyBy etc here too right?

Reply:
it's on the original order already